### ISSUE TITLE
Handle OSC messages without values

### ISSUE THREAD

**Issue by sensestage** [ModalityTeam/Modality-toolkit#186]
(ported over from #115)

One more case to cover:
osc messages that don't send a value, but are just a trigger:

e.g. messsages for different triggers:

/trigger 1
/trigger 2

Or two messages that basically make up one value:

/button/on
/button/off

**Comment by sensestage** [comment]
second one:
pattern match on
'/button/on'.patternMatch;
then use StringMap to find value.

**Comment by sensestage** [comment]
/trigger

```
(
trigger: (
	oscPath: '/trigger',
        valueAt: \none,
	type: 'trigger',
	ioType: \in
	)
]
)

**Comment by sensestage** [comment]
fixed in 3e72fbbfea9d3d92dc4eb72eee941398083805a3
